config/rbac/rbac_role.yaml

```diff
@@ -89,7 +89,6 @@ rules:
   - cluster.k8s.io
   resources:
   - machines
-  - machines/status
```

Comment:
It looks like this was previously added manually instead of through the kubebuilder annotations. Instead of removing this here, the annotation should be updated in: https://github.com/kubernetes-sigs/cluster-api/blob/master/pkg/controller/machine/controller.go#L93 to:

```go
// +kubebuilder:rbac:groups=cluster.k8s.io,resources=machines;machines/status,verbs=get;list;watch;create;update;patch;delete
```

Reply:
I have this fix as part of #857